tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/About.java

```diff
@@ -0,0 +1,103 @@
+package org.unicode.cldr.web.api;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import javax.servlet.ServletContext;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.unicode.cldr.util.CLDRConfigImpl;
+import org.unicode.cldr.util.CLDRFile;
+import org.unicode.cldr.util.CLDRURLS;
+import org.unicode.cldr.web.CookieSession;
+import org.unicode.cldr.web.DBUtils;
+import org.unicode.cldr.web.SurveyMain;
+
+import com.ibm.icu.util.VersionInfo;
+
+@Path("/about")
+public class About {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Get ST info",
+        description = "Returns detailed information about the Survey Tool")
+    @APIResponses(
+        value = {
+            @APIResponse(
+                responseCode = "200",
+                description = "Details about the Survey Tool",
+                content =  @Content(mediaType = "application/json",
+                schema = @Schema(type = SchemaType.OBJECT,
+                example = "{\n"
+                    + "  \"CLDR_COMMIT_BASE\": \"https://github.com/unicode-org/cldr/commit/\",\n"
+                    + "  \"CLDR_DATA_HASH\": \"f51da9c87d20b8d1236f0ea139a90ab49a879d65\",\n"
+                    + "  \"CLDR_SURVEYTOOL_HASH\": \"(unknown)\",\n"
+                    + "  \"CLDR_UTILITIES_HASH\": \"c27bb5ed340b63d4aa9fcb1ed948767d3a9869f3\",\n"
+                    + "  \"GEN_VERSION\": \"39\",\n"
+                    + "  \"ICU_VERSION\": \"68.1.0.0\",\n"
+                    + "  \"TRANS_HINT_LANGUAGE_NAME\": \"English\",\n"
+                    + "  \"TRANS_HINT_LOCALE\": \"en-ZZ\",\n"
+                    + "  \"java_vendor\": \"N/A\",\n"
+                    + "  \"java_version\": \"15.0.1\",\n"
+                    + "  \"java_vm_name\": \"OpenJDK 64-Bit Server VM\",\n"
+                    + "  \"java_vm_vendor\": \"Oracle Corporation\",\n"
+                    + "  \"java_vm_version\": \"15.0.1+9\",\n"
+                    + "  \"os_arch\": \"x86_64\",\n"
+                    + "  \"os_name\": \"Mac OS X\",\n"
+                    + "  \"os_version\": \"10.16\",\n"
+                    + "  \"serverInfo\": \"IBM WebSphere Liberty/21.0.0.1\",\n"
+                    + "  \"servletVersion\": \"4.0\"\n"
+                    + "}\n"
+                    + "")))})
+    public Response getAbout() {
+        Map<String,String> r = new TreeMap<>();
```

Comment:
for now it's a port (copy and paste really) of `AboutST.java` but could be more complex if needed.